patterns-modules/vertical-slice-architecture/src/main/java/com/baeldung/verticalslices/reader/ReadArticleUseCase.java

```diff
@@ -0,0 +1,19 @@
+package com.baeldung.verticalslices.reader;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.simple.JdbcClient;
+import org.springframework.stereotype.Component;
+
+@Component
+class ReadArticleUseCase {
+
+    @Autowired
+    private JdbcClient jdbcClient;
+
+    public void apply(Long id) {
+
+    }
+
+
+
+}
```

Comment:
Looks like formatting got a little weird here.

```suggestion
    }
}
```